src/helper_inbox.py

```diff
@@ -1,22 +1,25 @@
-"""Helper Inbox performs inbox messagese related operations."""
+"""Helper Inbox performs inbox messages related operations."""
 
-from helper_sql import sqlExecute, sqlQuery
 import queues
+from helper_sql import sqlExecute, sqlQuery
 
 
 def insert(t):
+	"""Perform an insert into the "inbox" table"""
     sqlExecute('''INSERT INTO inbox VALUES (?,?,?,?,?,?,?,?,?,?)''', *t)
     # shouldn't emit changedInboxUnread and displayNewInboxMessage
     # at the same time
     # queues.UISignalQueue.put(('changedInboxUnread', None))
 
 
 def trash(msgid):
+    """Mark a message in the `inbox` as `trash`"""
```

Comment:
@g1itch These backticks can force sphinx to treat inbox as object in near future?

Reply:
Yes. But I probably wont care about the sql-related helpers anymore. Because I'm going to replace them all when finish tests in #1534.